cypress/e2e/1-todo/todo.cy.js

```diff
@@ -0,0 +1,183 @@
+import 'cypress-real-events/support';
+
+describe('todo list app', () => {
+  it('할 일이 존재하지 않는다면 "There are no to-do items. Please write your to-dos" 가 나타난다.', () => {
+    cy.visit('http://localhost:5173');
+    cy.get('.todo-list-item').should('not.exist');
+    cy.get('.todo-empty-msg').should(
+      'contain',
+      'There are no to-do items. Please write your to-dos'
+    );
+  });
```

Comment:
p5. 전체적으로 `cy.visit('http://localhost:5173');` 이 코드가 중복되는 것 같은데, `todo list app` describe의 `beforeEach`로 푼다면 다른 곳에서 모두 제거 가능해보입니다!

```suggestion
describe('todo list app', () => { 
  beforeEach(() => {
    cy.visit('http://localhost:5173');
  });
  
  it('할 일이 존재하지 않는다면 "There are no to-do items. Please write your to-dos" 가 나타난다.', () => {
    cy.get('.todo-list-item').should('not.exist');
    cy.get('.todo-empty-msg').should(
      'contain',
      'There are no to-do items. Please write your to-dos'
    );
  });
```